tools/src/main/java/org/apache/kafka/tools/GetOffsetShell.java

```diff
@@ -283,6 +283,10 @@ private OffsetSpec parseOffsetSpec(String listOffsetsTimestamp) throws TerseExce
                 return OffsetSpec.latest();
             case "max-timestamp":
                 return OffsetSpec.maxTimestamp();
+            case "earliest-local":
```

Comment:
Could you please update docs?
https://github.com/apache/kafka/blob/2c9d7afe4c60656d10f5b15367e3fb65173d0372/tools/src/main/java/org/apache/kafka/tools/GetOffsetShell.java#L135